coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -162,14 +162,21 @@ def _extract_doc_comment(content, line, column, markers):
                     line, column and the extracted documentation. If not
                     matched, returns None.
     """
-    if markers[1] == '':
-        # Extract and align to start marker.
-        return _extract_doc_comment_simple(content, line, column, markers)
-    elif markers[1] == markers[2]:
-        # Search for the each-line marker until it runs out.
-        return _extract_doc_comment_continuous(content, line, column, markers)
-    else:
-        return _extract_doc_comment_standard(content, line, column, markers)
+    # This ignores fancy docstrings.
+    valid_docstring_regex = '\s*[' + markers[0] + '].*'
```

Comment:
@Makman2 
>[] defines a character class, not a group. Use () instead.

yup a group :O this is good.

>Actually I don't understand the regex. It will only match markers[0] in the worst case.

Ok so what im doing is that I find the valid docstrings. If you see `line` var. it contains only the starting line of the docstring. hence with `contain[line]` we get the first raw line. now regex only matches with `     """`. If theres something suppose `   abc="""xyz` on the first line. The regex returns `"""xyz` now I compare it with stripped first line(i.e `content[line]`) and voila as the strings doesn't match it is being ignored. The main reason of using `strip()` is, the regex I don't know why doesn't match the `\n` eg. if first raw line is `    """\n` regex matches `    """`.

Now I tried modifying begin regex but the problem I m facing is [here](https://github.com/damngamerz/coala/blob/e44c88d5b00728c12eac417ca37170c4f0a3f50e/coalib/bearlib/languages/documentation/DocumentationExtraction.py#L199)
regex will get spaces. Hence raising `KeyError` I can't use `strip()` as some markers have spaces in it.